test/common/README.md

```diff
@@ -53,23 +53,21 @@ Check if there is more than 1gb of total memory.
 ### expectsError([fn, ]settings)
 * `fn` [&lt;Function>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function)
 * `settings` [&lt;Object>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)
-  with the following optional properties:
-  * `code` [&lt;String>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type)
-    expected error must have this value for its `code` property
+  that may contain any optional property. Each property will be checked by
+  strict equality besides:
   * `type` [&lt;Function>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function)
-    expected error must be an instance of `type`
+    expected error must be an instance of `type` and should be [subclassed by
+    the `internal/errors` module](https://github.com/nodejs/node/blob/master/doc/guides/using-internal-errors.md#api).
   * `message` [&lt;String>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type)
     or [&lt;RegExp>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp)
     if a string is provided for `message`, expected error must have it for its
     `message` property; if a regular expression is provided for `message`, the
     regular expression must match the `message` property of the expected error
-
-* return function suitable for use as a validation function passed as the second
-  argument to `assert.throws()`
-
-If `fn` is provided, it will be passed to `assert.throws` as first argument.
-
-The expected error should be [subclassed by the `internal/errors` module](https://github.com/nodejs/node/blob/master/doc/guides/using-internal-errors.md#api).
+* return [&lt;Function>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function)
+  If `fn` is provided, it will be passed to `assert.throws` as first argument
```

Comment:
The `* Returns:` would be inconsistent with all other entries in this file. Shall I change all other entries as well?